ui-tests/src/test/java/org/openmrs/reference/page/RegistrationEditSectionPage.java

```diff
@@ -23,6 +23,7 @@
     private static final By STATE_PROVINCE = By.id("stateProvince");
     private static final By COUNTRY = By.id("country");
     private static final By POSTAL_CODE = By.id("postalCode");
+    private static final String PHONE_NUMBER_MESSAGE = "Must be a valid phone number (with +, -, numbers or parentheses))";
```

Comment:
PHONE_NUMBER_VALIDATION_MESSAGE would be clearer

Reply:
Thanks @dkayiwa  For reviewing and guidance. Resolved.